kuksa-appstore/build.sh

```diff
@@ -29,5 +29,5 @@ echo "##### Build app-store with Maven #####"
 mvn clean verify
 
 # Needs the xq command, see https://github.com/kislyuk/xq
```

Comment:
It seems the comment doesn't fit the code anymore

Reply:
true, I just removed the comment